src/protocol/mod.rs

```diff
@@ -67,6 +75,14 @@ impl GetRequest {
             ranges: RangeSpecSeq::all(),
         }
     }
+
+    /// Request just a single blob
+    pub fn just(hash: Hash) -> Self {
```

Comment:
I feel a bit stronger about bikeshedding here.  Not so much I care about my suggestion but I dislike this one a lot.  So I suggest `single`, it matches `all` fairly well.

Reply:
Yeah, will use single.